docs/guides/dependent-queries.md

```diff
@@ -1,20 +1,26 @@
 Dependent (or serial) queries depend on previous ones to finish before they can execute. To achieve this, it's as easy as using the `enabled` option to tell a query when it is ready to run:
 
 ```js
+// Main Query - get the user
+function useUserQuery(email) {
```

Comment:
I think it might confuse users, why every hook is wrapped in function.
Maybe additional page called similar to `good practices` could be added explaining why it is good to reuse your hooks with a wrapper, and explain it there?

Reply:
Agree, will add a page with the explanation

Reply:
Done